dso-l2/src/main/java/com/tc/l2/logging/StartupAndSizeBasedTriggeringPolicy.java

```diff
@@ -0,0 +1,17 @@
+package com.tc.l2.logging;
+
+import ch.qos.logback.core.rolling.SizeBasedTriggeringPolicy;
+
+import java.io.File;
+import java.util.concurrent.atomic.AtomicBoolean;
+
+public class StartupAndSizeBasedTriggeringPolicy<E> extends SizeBasedTriggeringPolicy<E> {
+    private final AtomicBoolean firstTime = new AtomicBoolean();
+
+    public boolean isTriggeringEvent(final File activeFile, final E event) {
+        if (firstTime.compareAndSet(false, true) && activeFile != null && activeFile.length() > 0) {
```

Comment:
Actually there isn't a case that I can see when the `activeFile` is null. So I guess those checks can be removed.